config/config/common-config.yaml

```diff
@@ -757,3 +757,6 @@ networkPolicies:
 
   dnsAutoscaler:
     enabled: true
+
+  alerts:
```

Comment:
I find this a litter confusing, as we do not actually deploy netpol to alerts
my proposal:
```
networkPolicies:
  enableAlerting: true
  # Default deny requires Calico as network plugin
  defaultDeny: false
  allowedNameSpaces: []
  additionalEgressPolicies: []
  additionalIngressPolicies: []
```